website/content/en/docs/about/under-the-hood/guarantees.md

```diff
@@ -15,13 +15,23 @@ see which components support specific guarantees.
 
 ### At-least-once
 
-The **at-least-once** delivery guarantee ensures that an [event] received by a Vector component is
-ultimately delivered at least once. While rare, it is possible for an event to be delivered more
-than once. See the [Does Vector support exactly-once delivery?](#faq-at-least-once) FAQ below).
+The **at-least-once** delivery guarantee ensures that an [event]
+received by a Vector component is ultimately delivered at least
+once. For a source, when the `acknowledgements` option is enabled, this
+indicates that it will wait for _all_ connected sinks to either mark the
+event as delivered or to persist the events to a durable buffer
+before acknowledging receipt of the event. For a sink, this indicates
+that it will attempt to retry the delivery until the events are either
+accepted or rejected and then signal the source with the results of that
+delivery.
```

Comment:
This was noted offline, but I think this conflates the transport delivery guarantees with acknowledgements. We mentioned having a separate field for sinks indicating acknowledgement support.